src/components/ScholarshipDetailCardCell.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles, Grid, Typography } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    color: 'rgba(0, 0, 0, 0.6)',
+    padding: `${theme.spacing(1.5)}px 0`,
+    borderBottom: '1px solid #EEEEEE',
+  },
+  labelWrapper: {
+    '& p': {
+      fontWeight: 'normal',
+    },
+  },
+  textWrapper: {
+    '& p': {
```

Comment:
What is this for?

Reply:
Not needed. Going to remove it and add the 'textAlign' to the Grid